.gitignore

```diff
@@ -1 +1,15 @@
 *.orig
+/skietbaan_fe/src/actions/types.js.orig
```

Comment:
just delete these the first entry should be fine to cover the `.orig` files
why are you ignoring normal things like `/skietbaan_fe/src/reducers/index.js`